content/theme/force/ways-of-the-force.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "The Ways of the Force"
+date: 2020-12-05
+draft: false
+---
+The Force represents one of the greatest sources of power in the galaxy; however, requires the will of a Force-sensitive being to effectively wield it, as technology is not
+a viable approach. To those with the appropriate sensitivity, it can be applied in a seemingly infinite variety of different ways. Individuals who are strong in the Force are 
+able to enhance their own abilities, understand their environment on a deeper or supernatural level, or even use it to impose their will upon their physical surroundings.
+
+#Learning About the Force
+However, this ability is quite rare. Further, though an individual may manifest sensitivity to the Force at any time, such a talent most be developed, rather than taught as a 
```

Comment:
```suggestion
This ability is quite rare. Further, though an individual may manifest sensitivity to the Force at any time, such a talent most be developed, rather than taught as a 
```